### ISSUE TITLE
beaver buider compat + better overview

### ISSUE THREAD

**Issue by ineagu** [Codeinwp/zerif-lite#574]
this task is the same for both zerif lite/pro and hestia, here is what we need to do : 

1. Make sure we have page-builder pages
2. add actions/hook for footer/header ( for beaver themer )
3. Add default style for various standard elements : https://www.wpbeaverbuilder.com/integrating-a-page-builder-with-your-wordpress-themes/
4. Create/design a new full beaverbuilder page ( about/ home alt )
5. Create custom modules if needed and have them added in companion ( alternatively we can use some modules from : https://wordpress.org/plugins/powerpack-addon-for-beaver-builder/ or https://wordpress.org/plugins/ultimate-addons-for-beaver-builder-lite/ )
6. Make the template available using this : http://kb.wpbeaverbuilder.com/article/107-theme-author-templates
7. Recommend beaver using TGM and set aff url using the filter in the first article to : https://www.wpbeaverbuilder.com/?fla=101&campaign=dash .

The logic here is more or less the same for : beaverbuilder, elementor or divi builder, here is how I see it working 👍 

- Focus on 2-3 and eventually 1 theme and build a as good as possible free version without relying on plugins
- Add basic css/compatibility for as many plugins as possible ( basic woo, 1 beaver template, etc )
- Add basic modules/widgets/mini-plugins in companion

then the pro theme version + premium companion will take care of 👍 

- style/templates for more advanced stuff ( different styles, 10-20 page samples, smarter woo style/checkout optimization, 5-10 different headers/footers, css for more niched plugins)
- extra modules/widgets/mini-plugins in companion

Basically we want to offer multi-purpose capabilities without theme-bloat and theme lock-in effects, which means, conditionally load css/php only when is needed.
I would recommend to have 1 person focus on a plugin ( for example @cristian-ungureanu deeply involved in beaver community, @abaicus in elementor etc ).

What is important is that the approach stays perfectly valid for the new gutenberg editor as well, when this is close to launch, we will just repeat the steps ( custom blocks, re-create pages etc ), so on the theming side the most valuable part will be UX/design/integrations. 

cc @rodica-andronache @bogdanpopa90 @stefan-cotitosu
